open-hackathon-server/src/hackathon/views/__init__.py

```diff
@@ -83,6 +82,7 @@ def init_routes():
     api.add_resource(TeamTemplateResource, "/api/team/template")  # select or unselect template for team
 
     # APIs for admin to manage hackathon and hackathon resources, features and users
+    api.add_resource(AdminHackathonTeamListResource, "/api/admin/team/list")  # list teams of hackathon
```

Comment:
`/api/admin/team/list` and `/api/hackathon/team/list` do similar things, the difference is that  `/api/admin/team/list` return more detail information about the team, and `/api/hackathon/team/list` is more secure.